spec/controllers/api_unlinks_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Api::UnlinksController do
+  describe "POST :create" do
+    context "if API token is incorrect" do
+      it "returns a 401 status code" do
+        redirection = Redirection.find_by!(slug: "gabe")
+
+        post :create, params: { redirection_slug: redirection.slug }
+
+        expect(response.status).to eq 401
+        expect(Redirection.find_by(slug: redirection.slug)).not_to be_nil
+        expect(BlockedReferrer.count).to eq 0
```

Comment:
Optional:

```suggestion
        expect(BlockedReferrer).to be_empty
```

At least I think that will work.

Reply:
Unfortunately it won't. It would have to be one of these:

```
expect(BlockedReferrer.any?).to be_true
```

```
# exists? does a fast count
expect(BlockedReferrer).not to_exist
```

I don't like either of these - `not to_exist` implies that the table is destroyed or something, and `any?` feels off.

What do you think?

Reply:
totally fair. I think it could also be `expect(BlockedReferrer.count).to be_zero` but I'm just bike-shedding at this point. What you wrote initially is clear and fine.